app/src/main/java/org/wikipedia/places/PlacesFragment.kt

```diff
@@ -695,5 +706,57 @@ class PlacesFragment : Fragment(), LinkPreviewDialog.LoadPageCallback, LinkPrevi
         fun latitudeDiffToMeters(latitudeDiff: Double): Int {
             return (111132 * latitudeDiff).toInt().coerceIn(10, 10000)
         }
+        fun showFeedbackOptionsDialog(activity: Activity) {
```

Comment:
Can this stuff be moved into its own file? Just so it will be easier to remove when we're done.